doc/user-manual/language/safe-agda.lagda.rst

```diff
@@ -8,48 +8,49 @@
+features leading to possible inconsistencies should be disabled.
 
 Here is a list of the features ``--safe`` is incompatible with:
 
-* ``postulate`` can be used to assume any axiom.
+* ``postulate``; can be used to assume any axiom.
 
-* ``--allow-unsolved-metas`` forces Agda to accept unfinished proofs.
+* ``--allow-unsolved-metas``; forces Agda to accept unfinished proofs.
 
-* ``--no-positivity-check`` makes it possible to write non-terminating
+* ``--no-positivity-check``; makes it possible to write non-terminating
   programs by structural "induction" on non strictly positive datatypes.
 
-* ``--no-termination-check`` gives loopy programs any type.
+* ``--no-termination-check``; gives loopy programs any type.
 
-* ``--type-in-type`` and ``--omega-in-omega`` allow the user to encode
+* ``--type-in-type`` and ``--omega-in-omega``; allow the user to encode
   the Girard-Hurken paradox.
 
-* ``--injective-type-constructors`` together with excluded middle leads
-  to an inconsistency via Chnug-Kil Hur's construction.
+* ``--injective-type-constructors``; together with excluded middle leads
+  to an inconsistency via Chung-Kil Hur's construction.
 
-* ``guardedness-preserving-type-constructors`` is based on a rather
-  operational understanding of ``∞``/``♯_``; it's not yet clear if
-  this extension is consistent.
+* ``--guardedness`` together with ``--sized-types``; currently can be
+  used to define a type which is both inductive and coinductive, which
+  leads to an inconsistency. This might be fixed in the future.
 
-* ``--experimental-irrelevance`` enables potentially unsound irrelevance
-  features (irrelevant levels, irrelevant data matching).
+* ``--experimental-irrelevance`` and ``--irrelevant-projections``;
+  enables potentially unsound irrelevance features (irrelevant levels,
+  irrelevant data matching, and projection of irrelevant record
+  fields, respectively).
 
-* ``--rewriting`` turns any equation into one that holds definitionally.
+* ``--rewriting``; turns any equation into one that holds definitionally.
   It can at the very least break convergence.
 
+* ``--cubical`` together with ``--with-K``; the univalence axiom is provable using cubical constructions, which falsifies the K axiom.
 
-Known Issues
-============
+* The ``primEraseEquality`` primitive together with ``--without-K``; using ``primEraseEquality``, one can derive the K axiom.
 
-Pragma Option
--------------
+The option ``--safe`` is coinfective (see
+:ref:`consistency-checking-options`); if a module is declared safe,
+then all its imported modules must also be declared safe.
 
-It is possible to specify ``{-# OPTIONS --safe #-}`` at the top of a file.
-Unfortunately a known bug (see `#2487 <https://github.com/agda/agda/issues/2487>`_)
-means that the option choice is not repercuted in the imported file. Therefore
-only the command-line option can be trusted.
+Known Issues
+============
 
 Standard Library
 ----------------
```

Comment:
This bit about the standard library can be removed now.

Reply:
Great, no more known issues! Fix incoming.